package/src/index.ts

```diff
@@ -0,0 +1,537 @@
+            data: { type: "matchdetails", value: matchID, region, queries },
+            method: "POST"
+        })
+    }
+
+    /**
+	 * Gets raw data for a players match history from the valorant API. (Not formatted, use only if you know what you are doing)
+	 * @see {@link default.getMatches} for an easier response to use
+	 * @param puuid - The match ID to get details about
+	 * @param region - Region of the player
+	 * @param queries - Extra queries
+	 * @returns Information about the players match history
+	 */
+    async getRawMatchHistory(puuid: string, region: Region, queries?: any): Promise<APIResponse<RawMatchHistoryResponse>> {
+        this._validateArgs({ puuid, region })
+        return await this._fetch<RawMatchHistoryResponse>("v1/raw", null, {
+            data: { type: "matchhistory", value: puuid, region, queries },
+            method: "POST"
+        })
+    }
+
+    /**
+	 * Gets raw data for a players competitive updates (rr changes) from the valorant API. **(Not formatted, use only if you know what you are doing)**
+	 * @see {@link default.getMMRHistory} for an easier response to use
+	 * @param puuid - The match ID to get details about
+	 * @param region - Region of the player
+	 * @param queries - Extra queries
+	 * @returns Information about the players rr history
+	 */
+    async getRawCompetitiveUpdates(puuid: string, region: Region, queries?: any): Promise<APIResponse<RawCompetitiveUpdatesResponse>> {
+        this._validateArgs({ puuid, region })
+        return await this._fetch<RawCompetitiveUpdatesResponse>("v1/raw", null, {
+            data: { type: "mmr", value: puuid, region, queries },
+            method: "POST"
+        })
+    }
+
+    /**
+	 * Gets raw data for a players mmr from the valorant API. **(Not formatted, use only if you know what you are doing)**
+	 * @see {@link default.getMMR} for an easier response to use
+	 * @param puuid - The match ID to get details about
+	 * @param region - Region of the player
+	 * @param queries - Extra queries
+	 * @returns Information about the players mmr
+	 */
+    async getRawMMR(puuid: string, region: Region, queries?: any): Promise<APIResponse<RawMMRResponse>> {
+        this._validateArgs({ puuid, region })
+        return await this._fetch<RawMMRResponse>("v1/raw", null, {
+            data: { type: "competitiveupdates", value: puuid, region, queries },
+            method: "POST"
+        })
+    }
+
+    /**
+	 * Get a list of rr changes of a player by their Riot ID
+	 * @param name - The Riot ID username of the player
+	 * @param tag - The Riot tag username of the player
+	 * @param region - The players region
+	 * @return List of rr changes (recent competitive games)
+	 */
+    async getMMRHistory(name: string, tag: string, region: Region): Promise<APIResponse<MMRHistoryResponse>> {
+        this._validateArgs({ name, tag, region })
+        return this._fetch<MMRHistoryResponse>(`v1/mmr-history/${region}/${name}/${tag}`)
```

Comment:
Maybe extract the endpoint definitions and write them into an extra file, see: https://github.com/raimannma/ValorantAPI/blob/master/valo_api/endpoints_config.py

Reply:
Maybe, although it seems just as easy to update them as new API changes come out. I'll do it ig